simulation.py

```diff
@@ -0,0 +1,73 @@
+"""
+A function to simulate alpha
+"""
+
+import copy
+import importlib
+import sys
+from thousandaire.constants import TRADING_CONFIG, DATA_LIST_ALL
+from thousandaire.data_loader import DataLoader
+from thousandaire.simulator import Simulator
+
+DATASET = 'price_dataset'
+
+def initialize():
+    """
+    Initialize data we get.
+    """
+    raw_data = DataLoader(DATA_LIST_ALL).get_all()
+    workdays = raw_data['workdays']
+    invest_data = {index : value
+                   for index, value in raw_data.items() if index != 'workdays'}
+    return_data = {}
+    for key, value in workdays.items():
+        return_data[key] = copy.deepcopy(invest_data)
+        for dataset in return_data[key].values():
+            dataset.set_workdays(value)
+    #Notice, this is a little different from other dataset in return_data.
+    return_data['workdays'] = workdays
+    return return_data
+
+def set_data(raw_data, data_list, target, region):
+    """
+    Use to filter data we need.
+    In the future may put in simulator to pickle needed data while
+    using multi-process.
+    """
+    workdays = raw_data['workdays'][region]
+    if target in raw_data[region].keys():
+        target_data = copy.deepcopy(raw_data[region][target])
+    else:
+        raise KeyError("Key not found")
+    simulate_data = {name : copy.deepcopy(raw_data[region][name])
+                     for name in data_list if name in raw_data[region].keys()}
+    data = {
+        'workdays' : workdays,
+        'target_data' : target_data,
+        'simulate_data' : simulate_data}
+    return data
+
+def simulation(alpha_path, raw_data):
+    """
+    Act the process of simulating.
+    """
+    setting_module = importlib.import_module(alpha_path).AlphaSettings()
```

Comment:
It is possible that the input value is invalid. Can you add a try-exception handler here?